addon/authenticators/jwt.js

```diff
@@ -176,11 +185,11 @@ export default BaseAuthenticator.extend({
   */
   _validate(data) {
     // Validate that a token is present
-    if (Ember.isEmpty(data['token'])) {
+    if (Ember.isEmpty(data[this.responseJwtName])) {
```

Comment:
this.get('responseJwtName') since it's an Ember property?

Reply:
Sure thing.  I was trying to make it consistent with the other properties in this file like `serverTokenEndpoint`.  Is there a way to do that, or should I just keep it as is and use `this.get(...)`?

Reply:
Never mind, I see how you're doing that.  Don't know how I missed that.